Scripts/Install/GTM/createVistaInstance.sh

```diff
@@ -194,11 +194,11 @@ echo "source $basedir/etc/env" >> $basedir/.bashrc
 gtmroutines="\$basedir/r/\$gtmver(\$basedir/r)"
 
 # 64bit GT.M can use a shared library instead of $gtm_dist
-if [[ $gtm_arch == "x86_64" && -e $basedir/lib/gtm/libgtmutil.so ]]; then
-    echo "export gtmroutines=\"$gtmroutines $basedir/lib/gtm/libgtmutil.so $basedir/lib/gtm\"" >> $basedir/etc/env
-else
-    echo "export gtmroutines=\"$gtmroutines $basedir/lib/gtm\"" >> $basedir/etc/env
-fi
+# if [[ $gtm_arch == "x86_64" && -e $basedir/lib/gtm/libgtmutil.so ]]; then
+    echo "export gtmroutines=\"$gtmroutines $basedir/lib/gtm/libgtmutil.so\"" >> $basedir/etc/env
```

Comment:
Should this be commented out too?

Reply:
No, the line here needs to remain to set the gtmroutines variable at all.  If anything, we should remove the indent to show that it doesn't rely on the if statement any more